.github/workflows/testing.yml

```diff
@@ -0,0 +1,23 @@
+name: simple_unit_tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - reopened
+    branches: 
+      - development
+      - master
+      
+jobs:
+  Run_Unit_Tests:
+    runs-on: windows-latest
```

Comment:
It's more common to use a Unix base image but this is also okay. I just have never used Windows for CI/CD and am not sure of what to expect especially when the codebase grows or when more workflows are added and we start to face certain issues

Reply:
Good point.  I put that as part of trouble shooting but it wasn't the problem. I decided to add both (for now) but because I can. 

I'm almost certain there is a more elegant way to do this (strategy: matrix) but I have some other learning objectives for the night and so I just implemented both jobs the nooby way.